.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+// Load Jenkins shared library
+jenkinsBranch = 'sift-android'
+sharedLib = library("shared-lib@${jenkinsBranch}")
+
+def siftAndroidWorkflow = sharedLib.com.sift.ci.SiftAndroidWorkflow.new()
+def ciUtil = sharedLib.com.sift.ci.CIUtil.new()
+def stackdriver = sharedLib.com.sift.ci.StackDriverMetrics.new()
+
+// Default GitHub status context for automatically triggered builds
+def defaultStatusContext = 'Jenkins:auto'
+
+// Pod template file for Jenkins agent pod
+// Pod template yaml file is defined in https://github.com/SiftScience/jenkins/tree/master/resources/jenkins-k8s-pod-templates
+def podTemplateFile = 'android-pod-template.yaml'
+def podLabel = "android-${BUILD_TAG}"
+
+
+
+// GitHub repo name
+def repoName = 'sift-android'
+
+pipeline {
+    agent none
+    options {
+        timestamps()
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+        disableRestartFromStage()
+        parallelsAlwaysFailFast()
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: '')
+        timeout(time: 30, unit: 'MINUTES')
+
+    }
+    environment {
+        GIT_BRANCH = "${env.CHANGE_BRANCH != null? env.CHANGE_BRANCH : env.BRANCH_NAME}"
+        ANDROID_SDK_ROOT = "/opt/android/sdk"
+    }
+    stages {
+        stage('Initialize') {
+            steps {
+                script {
+                    statusContext = defaultStatusContext
+                    // Get the commit sha for the build
+                    commitSha = ciUtil.commitHashForBuild()
+                    ciUtil.updateGithubCommitStatus(repoName, statusContext, 'Started', 'pending', commitSha)
+                }
+            }
+        }
+        stage ('Workflow') {
```

Comment:
There is no need for parallel workflows since it is simple pipeline.  This can be simplified like this.
```
stage('Build') {
            steps {
                script {
                    ciUtil.updateGithubCommitStatus(repoName, 'Build', 'Started', 'pending', commitSha)
                            try {
                                siftAndroidWorkflow.runSiftAndroidBuild(podTemplateFile, podLabel)
                                ciUtil.updateGithubCommitStatus(repoName, 'Build', 'SUCCESS', 'success', commitSha)
                            } catch (Exception e) {
                                ciUtil.updateGithubCommitStatus(repoName, 'Build', 'FAILURE', 'failure', commitSha)
                                print("${stage1} failed")
                                throw e
                            }
                }
            }
        }